### ISSUE TITLE
"X feeds synced" UX

### ISSUE THREAD

**Issue by aaronshaf** [beakerbrowser/fritter#43]
I'd recommend against the spinner to indicate loading user archives: https://cl.ly/1p02320w0P3r

It gives a nervous sense of awaiting completion. 

In my opinion, Dat apps on Beaker will have to make the inability to connect to an infrequently available Dat archive (say, another user one is following) less annoying and feel more acceptable. Occasional synchronization should be completely tolerable in the new P2P era.

**Comment by pfrazee** [comment]
Yeah I'm not happy with it right now, especially because it seems to have a common case where it never goes away. That said, I do like to see info about the sync state. Removing the spinner might also be smart, for the reason you suggest.
